dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/cslb/CslbExtractionPipeline.scala

```diff
@@ -3,11 +3,16 @@ package org.broadinstitute.monster.dap.cslb
 import org.broadinstitute.monster.common.{PipelineBuilder, ScioApp}
 import org.broadinstitute.monster.dap.common._
 
+import java.time.{OffsetDateTime, ZoneOffset}
+
 // Ignore IntelliJ, this is used to make the implicit parser compile.
 import Args._
+class CslbExtractionFailException() extends scala.Exception {}
```

Comment:
The parens and `scala.` are redundant here. I think you can do:

```suggestion
class CslbExtractionFailException extends Exception
```